### PR TITLE
Pop up unsupported dialog when Bluetooth and WebUSB are unsupported

### PR DESCRIPTION
**Pull request by microbit-grace**
https://microbit-global.monday.com/boards/1356069004/pulses/1372559402

### CODE REVIEW

File: src/pages/Homepage.svelte [hunk 1]
```diff
@@ -71,6 +85,7 @@
   <title>{title}</title>
 </svelte:head>
 
+<CompatibilityWarningDialog />
```
Comment (microbit-robert) [inline src/pages/Homepage.svelte:88]: We don't need this here as it's already in App.svelte.

File: src/components/LinkOverlay.svelte [hunk 2]
```diff
@@ -22,23 +22,34 @@
...
+  <a href={onClickOrHrefOrPath} on:click={handleClickPath} class={sharedClass}>
+    <slot />
+  </a>
+{:else if typeof onClickOrHrefOrPath === 'string'}
+  <a href={onClickOrHrefOrPath} class={sharedClass}>
+    <slot />
+  </a>
+{:else}
+  <!-- svelte-ignore a11y-no-static-element-interactions -->
+  <a on:click={onClickOrHrefOrPath} class={sharedClass}>
```
Comment (microbit-robert) [inline src/components/LinkOverlay.svelte:52]: I think this should be a button if there's no href.

**Review by microbit-robert — CHANGES_REQUESTED** [review]
Minor changes requested, but works well.

File: src/pages/Homepage.svelte (hunk 1 above)
Comment (microbit-grace) [reply src/pages/Homepage.svelte]: Ahh yes. https://github.com/microbit-foundation/ml-trainer/pull/185/commits/e2f17fed4eae392fe03939c0ae1186c9e9af9177

**Review by microbit-robert — APPROVED** [review]
LGTM, thanks.

**Pull request merged**